firefox-ios/firefox-ios-tests/Tests/XCUITests/JumpBackInTests.swift

```diff
@@ -162,18 +162,19 @@ class JumpBackInTests: BaseTestCase {
         waitUntilPageLoad()
         navigator.nowAt(BrowserTab)
         navigator.performAction(Action.GoToHomePage)
-        mozWaitForElementToExist(app.cells["JumpBackInCell"].firstMatch)
-        app.cells["JumpBackInCell"].firstMatch.press(forDuration: 2)
-        // The context menu opens, having the correct options
-        let ContextMenuTable = app.tables["Context Menu"]
-        waitForElementsToExist(
-            [
-                ContextMenuTable,
-                ContextMenuTable.cells.otherElements[StandardImageIdentifiers.Large.plus],
-                ContextMenuTable.cells.otherElements[StandardImageIdentifiers.Large.privateMode],
-                ContextMenuTable.cells.otherElements[StandardImageIdentifiers.Large.bookmark],
-                ContextMenuTable.cells.otherElements[StandardImageIdentifiers.Large.share]
-            ]
-        )
+        // After homepage redesign, "Jump back in" is no longer visible (FXIOS-11644)
+        // mozWaitForElementToExist(app.cells["JumpBackInCell"].firstMatch)
```

Comment:
The missing "Jump Back In" section is also a known issue.

Reply:
Jump Back In ticket: https://mozilla-hub.atlassian.net/browse/FXIOS-11644